charts/landscaper/charts/landscaper/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+name: landscaper
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: v0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: v0.13.0
```

Comment:
we have a automation that adjust this value for every release.
I'm not sure if this value is now still updated in the sub charts (which I think we have to?).
Can you check that

Reply:
This is done here: https://github.com/gardener/landscaper/blob/master/hack/create-helm-chart.sh#L45
Since we are also publishing the sub charts, the versions will also be updated.
I have checked that and it works as expected.